### DISCUSSION TITLE
Copy doesn't copy the right line

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#67780] [Mobile] [answered]
GitHub Mobile for Android beta

Visit:
https://github.com/jsoref/notifications-engine/actions/runs/6256946901/job/16988546546

Select the line you see here:
![Screenshot_20230920-235449.png](https://github.com/community/community/assets/2119212/c3c4e8ad-17bc-4ef3-abe3-b8e61c61c020)

Click the copy widget at the top right

This is copied to the clipboard (you can see it in the picture):
> 2023-09-21T03:50:21.9429110Z pkg/services/rocketchat.go:83:12 ... 17, Error - `Emoji` is not a recognized word. (unrecognized-spelling)

Expect the selected line to be copied:
https://github.com/jsoref/notifications-engine/actions/runs/6256946901/job/16988546546#step:2:410

**Comment by stevepopovich** [comment, marked as answer]
Hi @jsoref, I see this issue. I will report to the team. Thanks!
